pkg/ovssubnet/common.go

```diff
@@ -21,6 +21,8 @@ import (
 const (
 	// Maximum VXLAN Network Identifier as per RFC#7348
 	MaxVNID = ((1 << 24) - 1)
+	// VNID for the admin namespaces
+	VNID0 = uint(0)
```

Comment:
"AdminVNID"? If we think it might ever be something other than 0, then "VNID0" is a bad name, and if we don't think it might ever be non-0, then there's no reason to not just hardcode 0...